packages/api-reference/src/components/Anchor/Anchor.vue

```diff
@@ -21,60 +27,31 @@ const handleCopy = () => {
 }
 </script>
 <template>
-  <span class="label">
+  <span v-bind="cx('group/heading word-break-all relative')">
     <span
       :id="labelId"
       class="contents">
       <slot />
     </span>
-    <span class="anchor">
+    <span class="relative">
       <!-- Position anchor to align the copy button to the last line of text  -->
       <span>&ZeroWidthSpace;</span>
-      <button
+      <ScalarButton
         :aria-describedby="labelId"
-        class="anchor-copy"
-        type="button"
+        class="absolute left-0 inline-block h-fit px-1.5 py-1 opacity-0 group-hover/heading:opacity-100 group-has-focus-visible/heading:opacity-100"
+        :class="{
+          'top-1': layout === 'modern',
+        }"
```

Comment:
ideally we get the CSS straight without relying on the `layout`, can you try if you see a generic solution that doesn’t require a conditional class?

the few places where we have those seems to have an even higher risk for UI regressions it seems

Reply:
only ui regression or related to lack of testing both IMO so this won't change it but can look to rely on a class, that will still rely on the layout at the end of the day i think. will dig!